### PR TITLE
ManualSetup doc: cover interfering software

### PR DESCRIPTION
**Pull request by neiljerram**

### CODE REVIEW

File: docs/calico-with-docker/without-docker-networking/ManualSetup.md [hunk 1]
```diff
@@ -18,8 +18,14 @@ the example.
 
 ## Requirements
 
-You will need 2 servers (bare metal or VMs) with a  modern 64-bit Linux OS
-and IP connectivity between them.
+You will need 2 servers (bare metal or VMs) with a modern 64-bit Linux OS and
+IP connectivity between them.  The servers must not have any software running
+that tries to manage or interfere with new interfaces that appear (for example
+adding or removing routes or iptables through them, or modifying interface
+state), as that would fundamentally conflict with Calico's programming.  We
```
Comment (lwr20) [inline docs/calico-with-docker/without-docker-networking/ManualSetup.md]: "would fundamentally conflict with Calico's programming".  

Perhaps "is likely to conflict with Calico's programming of iptables and ipsets, leading to loss of connectivity with endpoints"?

Comment (neiljerram) [inline docs/calico-with-docker/without-docker-networking/ManualSetup.md]: Well it's not just iptables/sets, it's routing too; but the other changes sound good to me.  I'll push a delta.

**Comment by neiljerram** [comment]
Ready for another look?

File: docs/calico-with-docker/without-docker-networking/ManualSetup.md [hunk 2]
```diff
@@ -18,8 +18,14 @@ the example.
 
 ## Requirements
 
-You will need 2 servers (bare metal or VMs) with a  modern 64-bit Linux OS
-and IP connectivity between them.
+You will need 2 servers (bare metal or VMs) with a modern 64-bit Linux OS and
+IP connectivity between them.  The servers must not have any software running
+that tries to manage or interfere with new interfaces that appear (for example
+adding or removing routes or iptables through them, or modifying interface
```
Comment (lwr20) [inline docs/calico-with-docker/without-docker-networking/ManualSetup.md]: I'm worried about "through them" - it didn't mean to me what it means to you, and I'm a native english speaker.

Perhaps: "The servers must not run any software that tries to manage or interfere with new interfaces or related config (for example routes, iptables, ipsets or interface state)"

**Comment by neiljerram** [comment]
And again...

**Comment by lwr20** [comment]
LGTM

**Pull request merged**